package/utils/busybox/config/networking/Config.in

```diff
@@ -9,7 +9,6 @@ menu "Networking Utilities"
 config BUSYBOX_CONFIG_FEATURE_IPV6
 	bool "Enable IPv6 support"
 	default BUSYBOX_DEFAULT_FEATURE_IPV6
-	depends on IPV6
```

Comment:
It makes no sense to remove the dependency on IPV6 as it makes BUSYBOX_CONFIG_FEATURE_IPV6 configurable for the user while that should not be the case if IPV6 is diabled